parse/BuildingsParser.cpp

```diff
@@ -48,21 +40,23 @@ namespace {
             qi::_b_type _b;
             qi::_c_type _c;
             qi::_d_type _d;
+            qi::_pass_type _pass;
             qi::_r1_type _r1;
             qi::eps_type eps;
 
             const parse::lexer& tok = parse::lexer::instance();
 
             building_type
                 =   tok.BuildingType_
-                >   parse::detail::label(Name_token)                > tok.string        [ _a = _1 ]
+                >   parse::detail::label(Name_token)
+                >   tok.string        [ _pass = is_unique_(_r1, "BuildingType", _1), _a = _1 ]
```

Comment:
Thanks for finding and reporting this.  

It appears to be losing the constness of the string literal while trying to convert it to `const std::string`.  I've changed the parameter of `is_unique` to `const char* const`, which hopefully prevents it from trying to convert anything.

I also switched all of the literals to token, which they should have been in the first place.